pkg/test/echo/cmd/client/main.go

```diff
@@ -162,12 +167,21 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&serverName, "server-name", "", serverName, "server name to set")
 
 	rootCmd.PersistentFlags().StringVar(&hboneAddress, "hbone", "", "address to send HBONE request to")
+	rootCmd.PersistentFlags().StringVar(&doubleHboneAddress, "double-hbone", "", "address to send double HBONE request to")
 	rootCmd.PersistentFlags().StringSliceVarP(&hboneHeaders, "hbone-header", "M", hboneHeaders,
 		"A list of http headers for HBONE connection (use Host for authority) - 'name: value', following curl syntax")
 	rootCmd.PersistentFlags().StringVar(&hboneCaFile, "hbone-ca", "", "CA root cert file used for the HBONE request")
 	rootCmd.PersistentFlags().StringVar(&hboneClientCert, "hbone-client-cert", "", "client certificate file used for the HBONE request")
 	rootCmd.PersistentFlags().StringVar(&hboneClientKey, "hbone-client-key", "", "client certificate key file used for the HBONE request")
 	rootCmd.PersistentFlags().BoolVar(&hboneInsecureSkipVerify, "hbone-insecure-skip-verify", hboneInsecureSkipVerify, "skip TLS verification of HBONE request")
+	rootCmd.PersistentFlags().StringVar(&innerHboneCaFile, "inner-hbone-ca", "",
+		"CA root cert file used for the inner HBONE request. Only used if --double-hbone is set")
+	rootCmd.PersistentFlags().StringVar(&innerHboneClientCert, "inner-hbone-client-cert", "",
```

Comment:
I don't mind it but its probably not something we will ever need to customize; double hbone should use the same creds for inner and outer?

maybe the CA and insecure but key/cert seems odd

Reply:
Ack, agreed - just CA and insecure